app/services/ccms/submitters/obtain_document_id_service.rb

```diff
@@ -48,7 +52,7 @@ def request_document_ids
       end
 
       def document_id_requestor
-        @document_id_requestor ||= CCMS::Requestors::DocumentIdRequestor.new(submission.case_ccms_reference, submission.legal_aid_application.provider.username)
+        @document_id_requestor ||= Requestors::DocumentIdRequestor.new(submission.case_ccms_reference, submission.legal_aid_application.provider.username)
```

Comment:
you could use the above #legal_aid_application method to get the provider username e.g. `legal_aid_application.provider.username`